### PR TITLE
Add webview for AppMap Navie

### PR DESCRIPTION
**Pull request by jansorg**
Part of https://github.com/getappmap/appmap-intellij-plugin/issues/533

This adds the assets for the Navie webview. The webview is not yet used or displayed by the plugin.

### CODE REVIEW

File: appland-navie/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "publisher": "appland",
+  "name": "appmap-intellij-install-guide",
```
Comment (ahtrotta) [inline appland-navie/package.json:3]: This should be `appmap-intellij-navie`

File: appland-navie/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        }
+    </style>
+</head>
+<body>
+<div id="app"></div>
+<script src="./dist/main.js"></script>
+<script type="text/javascript">
+    AppLandWeb.mountWebview();
+</script>
+</body>
```
[the remark this answers is not in the document]
Comment (jansorg) [reply appland-navie/index.html:22]: Done (force-pushed)

File: appland-navie/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "0.0.1",
+  "packageManager": "yarn@4.0.2",
```
[the remark this answers is not in the document]
Comment (jansorg) [reply appland-navie/package.json]: Done (force-pushed). I wanted to update the others to the current version 4.0.2, but it's better to use the same value here now.

File: appland-navie/package.json (hunk 1 above)
Comment (jansorg) [reply appland-navie/package.json]: Done (force-pushed)

**Review by ahtrotta — APPROVED** [review]

**Pull request merged**